src/api.c

```diff
@@ -50,6 +50,12 @@
 #include <linux/un.h>
 #include <grp.h>
 
+const struct cgroup_library_version library_version = {
+	.major = CGROUP_VER_MAJOR,
+	.minor = CGROUP_VER_MINOR,
+	.release = CGROUP_VER_RELEASE,
+};
+
```

Comment:
The `configure.ac`, has the following values for the library versions:
```# set library version, soname is libcgroup.so.MAJOR
AC_SUBST(LIBRARY_VERSION_MAJOR, 1)
AC_SUBST(LIBRARY_VERSION_MINOR, 0)
AC_SUBST(LIBRARY_VERSION_RELEASE, 42)
```
 Do those need to be updated to `2.0.0`?

Reply:
Yes, but I think we should mimic what libseccomp has done here.  In the main branch's [configure.ac](https://github.com/seccomp/libseccomp/blob/main/configure.ac#L22), the version is set to `0.0.0`.  This way anything built with the main branch will have a strange version number that should signify to a user that it's not an official release.  In the release branch, then we can set the version to the appropriate number.  As you pointed out, `2.0.0` or whatever.

Reply:
Interesting idea ! we should do the same.